charts/argo-cd/values.yaml

```
@@ -1752,6 +1654,258 @@ repoServer:
+      #     x15OklctmrZtqre5kwPUosG3/B2/ikuPYElcHgGPL4uL5Em6S5C/oozfkYzhwRrT
+      #     SQzvYjsE4I34To4UdE9KA97wrQjGoz2Bx72WDLyWwctD3DKQtYeHXswXXtXwKfjQ
+      #     7Fy4+Bf5IPh76dA8NJ6UtjjLIDlKqdxLW4atHe6xWFaJ+XdLUtsAroZcXBeWDCPa
+      #     buXCDscJcLJRKZVc62gOZXXtPfoHqvUPp3nuLA4YjH9bphbrMWMf810Wxz9JTd3v
+      #     yWgGqNY0zbBqeZoGv+TuExlRHT8ASGFS9SVDABEBAAG0NUdpdEh1YiAod2ViLWZs
+      #     b3cgY29tbWl0IHNpZ25pbmcpIDxub3JlcGx5QGdpdGh1Yi5jb20+iQEiBBMBCAAW
+      #     BQJZlGhBCRBK7hj4Ov3rIwIbAwIZAQAAmQEH/iATWFmi2oxlBh3wAsySNCNV4IPf
+      #     DDMeh6j80WT7cgoX7V7xqJOxrfrqPEthQ3hgHIm7b5MPQlUr2q+UPL22t/I+ESF6
+      #     9b0QWLFSMJbMSk+BXkvSjH9q8jAO0986/pShPV5DU2sMxnx4LfLfHNhTzjXKokws
+      #     +8ptJ8uhMNIDXfXuzkZHIxoXk3rNcjDN5c5X+sK8UBRH092BIJWCOfaQt7v7wig5
+      #     4Ra28pM9GbHKXVNxmdLpCFyzvyMuCmINYYADsC848QQFFwnd4EQnupo6QvhEVx1O
+      #     j7wDwvuH5dCrLuLwtwXaQh0onG4583p0LGms2Mf5F+Ick6o/4peOlBoZz48=
+      #     =Bvzs
+      #     -----END PGP PUBLIC KEY BLOCK-----
+
+  # Argo CD rbac config
+  ## Ref: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/rbac.md
+  rbac:
+    # -- Create argocd-rbac-cm ConfigMap.
+    # If false, it is expected that the configmap will be created by something else.
+    # Argo CD will not work if this configuration is missing.
+    enabled: true
+
+    # -- Annotations to be added to argocd-rbac-cm ConfigMap
+    annotations: {}
+
+    policy:
+      # -- Default role which Argo CD will fall back to, when authorizing API requests
+      default: role:readonly
+      # -- User defined [Argo CD RBAC policy] rules
+      rules: ""
+      # rules: |
+      #   # Policy rules are in the form
+      #   p, subject, resource, action, object, effect
+      #   # Role definitions and bindings are in the form:
+      #   g, subject, inherited-subject
+    # -- Scopes controls which OIDC scopes to examine during rbac enforcement
+    scopes:
+      - groups
+      # - email
+      # - cognito:groups
+
+  # Argo CD secret
+  ## Ref: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-secret.yaml
+  secret:
+    # -- Create the argocd-secret
+    enabled: true
+    # -- Annotations to be added to argocd-secret
+    annotations: {}
+
+    # -- Shared secret for authenticating GitHub webhook events
+    githubSecret: ""
+    # -- Shared secret for authenticating GitLab webhook events
+    gitlabSecret: ""
+    # -- Shared secret for authenticating BitbucketServer webhook events
+    bitbucketServerSecret: ""
+    # -- UUID for authenticating Bitbucket webhook events
+    bitbucketUUID: ""
+    # -- Shared secret for authenticating Gogs webhook events
+    gogsSecret: ""
+
+    # -- add additional secrets to be added to argocd-secret
+    ## Custom secrets. Useful for injecting SSO secrets into environment variables.
+    ## Ref: https://argoproj.github.io/argo-cd/operator-manual/sso/
+    ## Note that all values must be non-empty.
+    extra:
+      {}
```

```suggestion
    extra: {}
```

What about unify how empty dicts are defined across this file?

---

Done